src/code/z_scene.c

```diff
@@ -343,67 +343,63 @@ void func_80098CC8(GlobalContext* globalCtx, SceneCmd* cmd) {
 // Scene Command 0x11: Skybox Settings
 void func_80098D1C(GlobalContext* globalCtx, SceneCmd* cmd) {
     globalCtx->skyboxId = cmd->skyboxSettings.skyboxId;
-    globalCtx->envCtx.gloomySky = globalCtx->envCtx.unk_18 = cmd->skyboxSettings.unk_05;
-    globalCtx->envCtx.unk_1E = cmd->skyboxSettings.unk_06;
+    globalCtx->envCtx.unk_17 = globalCtx->envCtx.unk_18 = cmd->skyboxSettings.unk_05;
+    globalCtx->envCtx.indoors = cmd->skyboxSettings.unk_06;
 }
 
 // Scene Command 0x12: Skybox Disables
 void func_80098D5C(GlobalContext* globalCtx, SceneCmd* cmd) {
-    globalCtx->envCtx.skyDisabled = cmd->skyboxDisables.unk_04;
+    globalCtx->envCtx.skyboxDisabled = cmd->skyboxDisables.unk_04;
     globalCtx->envCtx.sunMoonDisabled = cmd->skyboxDisables.unk_05;
 }
 
 // Scene Command 0x10: Time Settings
 void func_80098D80(GlobalContext* globalCtx, SceneCmd* cmd) {
-    u32 dayTime;
-
     if ((cmd->timeSettings.hour != 0xFF) && (cmd->timeSettings.min != 0xFF)) {
-        gSaveContext.environmentTime = gSaveContext.dayTime =
-            ((cmd->timeSettings.hour + (cmd->timeSettings.min / 60.0f)) * 60.0f) / (360.0f / 0x4000);
+        gSaveContext.skyboxTime = gSaveContext.dayTime =
+            ((cmd->timeSettings.hour + (cmd->timeSettings.min / 60.0f)) * 60.0f) / 0.021972656f;
```

Comment:
Why get rid of `(360.0f / 0x4000)` ? (though `((f32)(24 * 60) / 0x10000)` would make more sense)

Reply:
I didnt change this myself, its just an old branch that must not have updated properly at some point during a merge. same case for a couple of these other things in this review
but yea thanks, will fix